### PR TITLE
Rename baas -> k8up in manifests

### PR DESCRIPTION
**Pull request by bittner**
Replaces the mentions of `baas` by `k8up` in the manifest files.

**Relates to:** #99

### CODE REVIEW

File: manifest/examples/prometheus/rules.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 groups:
 - name: K8up
   rules:
-  - alert: baas_last_errors
-    expr: baas_backup_restic_last_errors > 0
+  - alert: k8up_last_errors
```
Comment (Kidswiss) [inline manifest/examples/prometheus/rules.yml:4]: This may break the example configuration, as the prometheus metrics are still using baas.

Comment (bittner) [reply manifest/examples/prometheus/rules.yml:4]: Does this relate to the entire change set or only to these two lines?

What would you suggest? Should we close the entire PR, or only undo this specific change?

Comment (Kidswiss) [reply manifest/examples/prometheus/rules.yml:4]: It's only those two lines.

Comment (bittner) [reply manifest/examples/prometheus/rules.yml]: I undid the changes on those two lines.

**Comment by bittner** [comment]
@Kidswiss Shall we merge these changes now? Can you take another look?

**Pull request merged**